simple_history/tests/tests/utils.py

```diff
@@ -15,3 +15,13 @@
             settings.MIDDLEWARE_CLASSES + [request_middleware]
         )
     }
+
+
+def add_history_ip_address(sender, **kwargs):
+    history_instance = kwargs['history_instance']
+    if history_instance.question == 'read IP from request':
+        from simple_history.models import HistoricalRecords
+        history_instance.ip_address = \
+            HistoricalRecords.thread.request.META['REMOTE_ADDR']
```

Comment:
This differs from the `threading.local()` use that you have in docs. Did that not work? Should definitely update docs then